.circleci/config.yml

```diff
@@ -96,6 +96,7 @@ jobs:
      steps:
        - run: rm -rf /home/circleci/project/.git # CircleCI git caching is likely broken
        - checkout
+       - run: pip install -r tools/requirements.txt
```

Comment:
it seems weird to call this file requirements.txt -- is that a standard name for a file that lists Python library dependencies?

Reply:
Yeah, this is a standard for [pip](https://pip.pypa.io/en/stable/user_guide/#requirements-files)